en/docs/develop/components/rest-api.md

```diff
@@ -14,19 +14,19 @@ Choreo’s low-code editor allows developers to design (and develop) high-qualit
 
 Choreo allows you to design your REST API in three possible ways as shown in the image below:
 
-![Create REST API](../../assets/img/rest-apis/create-a-rest-api.png){.cInlineImage-half}
+![Create REST API](../../assets/img/rest-apis/create-a-rest-api.png){.cInlineImage-small}
```

Comment:
This is outdated. Created a separate issue [1] for this.

[1] https://github.com/wso2-enterprise/choreo/issues/18286